modules/installation-about-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+
+[id="installation-about-custom-gcp-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing Virtual Private Cloud (VPC) in Google Cloud Platform (GCP). If you do, you must also use existing subnets within the VPC and routing rules.
+
+By deploying {product-title} into an existing GCP VPC, you might be able to avoid service limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC.
+
+[id="installation-about-custom-gcp-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+When you deploy a cluster by using an existing VPC, you must perform additional network configuration before you install the cluster. In installer-provisioned infrastructure clusters, the installer usually creates the following components, but it does not create them when you install into an existing VPC:
+
+* VPC
+* Subnets
+* Cloud Router
+* Cloud NAT
+* NAT IP addresses
+
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+
+Your VPC and subnets must meet the following characteristics:
+
+* The VPC’s CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines.
```

Comment:
@kalexand-rh Based on the [google doc](https://cloud.google.com/vpc/docs/vpc), VPC networks do not have any IP address ranges associated with them. IP ranges are defined for the subnets. So this sentence means subnets CIDR must contain the `Networking.MachineCIDR` range which is conflict with below sentence "The subnet CIDRs belong to the machine CIDR". 
BTW, could you please let me know what is machine CIDR used for?
Thanks

Reply:
@kalexand-rh I do not find any updates for this comment.